### PR TITLE
Minimal header fix

### PR DESCRIPTION
**Pull request by melanierichards**

### CODE REVIEW

**Comment by melanierichards** [comment]
@molant this is marked as a failure but it looks like it's all flagged inside legacy code and not the new stuff. Mind if I merge it into master?

**Comment by ststimac** [comment]
@melanierichards @molant does the speech synth demo still need to be fixed? I know what code can be removed from my CSS file so it doesn't interfere with the demo header. 

edit: nevermind, talked to anton.

**Comment by melanierichards** [comment]
@ststimac we just removed the demo header for now, and some things that were borked were actually from changes to the demo-header.css file, not your demo. So don't worry about it :]

File: betafish/Default.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
+        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
```
Comment (molant) [inline betafish/Default.html]: Copyright character looks incorrect, can you just discard changes?

File: chalkboard/Default.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
     <head>
-        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
+        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
```
Comment (molant) [inline chalkboard/Default.html]: Copyright character looks incorrect, can you just discard changes?

File: fishbowlie/index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
     <head>
-        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
+        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
```
Comment (molant) [inline fishbowlie/index.html]: Copyright character looks incorrect, can you just discard changes?

File: demo-header.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
+/*
```
Comment (molant) [inline demo-header.css]: Do we need this file here or can we just use the one in the CDN now?

File: penguinmark/Default.html [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
+        <!-- Copyright � Microsoft Corporation. All Rights Reserved. -->
```
Comment (molant) [inline penguinmark/Default.html]: Copyright character looks incorrect, can you just discard changes?

**Review by molant — CHANGES_REQUESTED** [review]

**Review by molant — APPROVED** [review]

**Pull request merged**